datalad/support/sshconnector.py

```diff
@@ -118,6 +119,28 @@ def close(self):
             else:
                 raise
 
+    def copy(self, source, destination):
+        """Copies source file/folder to destination on the remote.
+
+        Parameters
+        ----------
+        source: str
+          file/folder path to copy from on local
+        destination: str
+          file/folder path to copy to on remote
+
+        Returns
+        -------
+        str
+          stdout, stderr of the copy operation.
+        """
+
+        self.scp_prefix = ["scp", "-o", "ControlPath=" + self.ctrl_path]
+        if isdir(source):  # add recursive flag if directory to be copied
+            self.scp_prefix = self.scp_prefix + ["-r"]
+        scp_cmd = self.scp_prefix + [source, self.host + ":" + destination]
```

Comment:
Where exactly? ;-) .host is only the hostname

On July 12, 2016 6:20:20 PM EDT, Debanjum <notifications@github.com> wrote:
>> +        ----------
>> +        source: str
>> +          file/folder path to copy from on local
>> +        destination: str
>> +          file/folder path to copy to on remote
>> +
>> +        Returns
>> +        -------
>> +        str
>> +          stdout, stderr of the copy operation.
>> +        """
>> +
>> +        self.scp_prefix = ["scp", "-o", "ControlPath=" +
>self.ctrl_path]
>> +        if isdir(source):  # add recursive flag if directory to be
>copied
>> +            self.scp_prefix = self.scp_prefix + ["-r"]
>> +        scp_cmd = self.scp_prefix + [source, self.host + ":" +
>destination]
>
>doesn't [this
>line](https://github.com/datalad/datalad/blob/5cb3a91528b2f525ba85d4a986166a52db40ca3a/datalad/support/sshconnector.py#L158)
>already handle setting the ssh port ?
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/datalad/datalad/pull/633/files/c38397b18cc79b9021c7cdf22d1989588c88a51f#r70533174